dockstore-integration-testing/src/test/java/io/dockstore/webservice/SwaggerWebhookIT.java

```diff
@@ -176,14 +176,22 @@ void testLambdaEvents() {
         final String dockstoreTestUser = "DockstoreTestUser";
         assertTrue(userOrganizations.contains(dockstoreTestUser)); // User has access to only one repo in the org, DockstoreTestUser/dockstore-whalesay-2
 
-        assertEquals(0, lambdaEventsApi.getLambdaEventsByOrganization(dockstoreTestUser, "0", 10).size(), "No events at all works");
+        assertEquals(0, lambdaEventsApi.getLambdaEventsByOrganization(dockstoreTestUser, 0, 10, null, null, null).size(), "No events at all works");
```

Comment:
This is up to you, but I have the urge to move this OpenAPI `testLambdaEvents` test to `WebhookIT` which only uses OpenAPI :slightly_smiling_face:. That way you can remove the package names like `io.dockstore.openapi.client`